### PR TITLE
Add params and literals methods to InsertBuilder, UpdateBuilder and DeleteBuilder

### PR DESCRIPTION
**Pull request by lunch2000**
#679

Please consider merging them if this request is not against doma's policy.

### CODE REVIEW

File: doma-core/src/test/java/org/seasar/doma/jdbc/builder/DeleteBuilderTest.java [hunk 1]
```diff
@@ -49,4 +55,78 @@ public void testLiteral() {
 
     builder.execute();
   }
+
+  @Test
+  public void testParams() throws Exception {
+    UpdateBuilder builder = UpdateBuilder.newInstance(new MockConfig());
```
Comment (nakamura-to) [inline doma-core/src/test/java/org/seasar/doma/jdbc/builder/DeleteBuilderTest.java:61]: I think `DeleteBuilder` is correct in this test class.

**Review by nakamura-to — CHANGES_REQUESTED** [review]
Thanks for your PR.
 I think it it almost good.

 But could you format your code?
 To format code, just run `./gradlew build`.

File: doma-core/src/test/java/org/seasar/doma/jdbc/builder/DeleteBuilderTest.java (hunk 1 above)
Comment (lunch2000) [reply doma-core/src/test/java/org/seasar/doma/jdbc/builder/DeleteBuilderTest.java:61]: I apologize.
I just fixed now.

**Comment by lunch2000** [comment]
Thank you for your review!

**Pull request merged**